update-nixpkgs.py

```diff
@@ -112,7 +112,7 @@ def rebase_nixpkgs(nixpkgs_repo: Repo, nixos_version: NixOSVersion):
 
 def prefetch_nixpkgs(nixos_version: str) -> dict[str, str]:
     prefetch_cmd = [
-        "nix-prefetch-github",
+        "/run/current-system/sw/bin/nix-prefetch-github",
```

Comment:
This assumes that the script is a system package, which is the case on our hydra right now. If we ever adopt the cleaner approach of just setting up a temporary nix shell at the target with the required packages, this needs to be changed back again.
So okay for now, despite it being not the cleanest way.